sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/TypeScriptPlugin.java

```diff
@@ -73,6 +78,15 @@ public void define(Context context) {
         .category(TYPESCRIPT_CATEGORY)
         .multiValues(true)
         .build(),
+      PropertyDefinition.builder(TSLINT_REPORT_PATHS)
+        .defaultValue(TSLINT_REPORT_PATHS_DEFAULT_VALUE)
+        .name("TSLint Result Files")
```

Comment:
why "result" and not "report"?

Reply:
@saberduck you are right, I will change it. TSLint docs uses "result", so I stuck to this word unconsciously :)